packages/devops/scripts/lambda/resources/startup.sh

```diff
@@ -60,7 +60,6 @@ git reset --hard origin/${BRANCH_TO_USE}
 # Deploy each package, including injecting environment variables from LastPass
 sudo -Hu ubuntu $DEPLOYMENT_SCRIPTS/fetchEnvVarsFromLastpass.sh $DEPLOYMENT_NAME |& while IFS= read -r line; do printf '\%s \%s\n' "$(date)" "$line"; done  >> $LOGS_DIR/deployment_log.txt
 sudo -Hu ubuntu $DEPLOYMENT_SCRIPTS/buildDeployablePackages.sh $DEPLOYMENT_NAME |& while IFS= read -r line; do printf '\%s \%s\n' "$(date)" "$line"; done  >> $LOGS_DIR/deployment_log.txt
-sudo -Hu ubuntu $DEPLOYMENT_SCRIPTS/setRDSMasterPassword.sh tupaia-$DEPLOYMENT_NAME |& while IFS= read -r line; do printf '\%s \%s\n' "$(date)" "$line"; done  >> $LOGS_DIR/deployment_log.txt
```

Comment:
No longer needed as we now handle this via the initial `spin_up_tupaia_deployment` lambda script